src/macaron/slsa_analyzer/git_url.py

```diff
@@ -536,9 +536,14 @@ def get_remote_vcs_url(url: str, clean_up: bool = True) -> str:
     str
         The remote url to the repo or empty if the url is invalid.
     """
+    if len(url) == 1:
```

Comment:
I wonder in which case this happens and why you need to check for that case here.

Reply:
You are right, I will investigate it more.